.github/workflows/tests.yml

```diff
@@ -41,4 +42,10 @@ jobs:
           git ls-files '**/*.php' | while read f; do php -l "$f" > /dev/null || touch failed && echo -n .; done
           test ! -f failed
 
+      # https://github.com/zf1s/zf1/pull/6#issuecomment-495397170
+      - name: Validate composer.json is valid for all packages
```

Comment:
There isn't probably a point to do `composer validate` with each PHP version. but having it this way keeps the CI pipeline simple. So wdyt?

Reply:
it's fine, thanks